src/main/java/com/google/starfish/servlets/HandleNotesServlet.java

```diff
@@ -18,16 +18,35 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;  
+import javax.servlet.http.Cookie;  
 
-@WebServlet("/upload-notes")
-public class UploadNotesServlet extends HttpServlet {
+@WebServlet("/handle-notes")
+public class HandleNotesServlet extends HttpServlet {
 
   private BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+  private final String COOKIE_NAME = "SFCookie";
   
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String blobKey = getUploadedFileBlobKey(request, "notes-pdf");
-    // TODO: Put blobKey and other course note information into database
+    
+    if(!validateUser(request)) {
+      response.setStatus(HttpServletResponse.SC_FORBIDDEN);
+      return;
+    }
+
+    HttpSession activeSession = request.getSession(false);
+    String userId = (String) activeSession.getAttribute("user_id");
+
+    String blobKey = getUploadedFileBlobKey(request, "file");
+    String title = request.getParameter("title");
+    String school = request.getParameter("school");
+    String course = request.getParameter("course");
+    // TODO: Not sure how the labels will be separated, will need to parse into array of strings
+    String miscLabels = request.getParameter("miscLabels");
```

Comment:
Hopefully [getParameterValues()](https://www.java4s.com/java-servlet-tutorials/example-on-getparametervalues-method-of-servlet-request/) will work.
```suggestion
    // TODO: Not sure how the labels will be separated, will need to parse into array of strings
    String[] miscLabels = request.getParameterValues("miscLabels");
```

Reply:
I tried out getParameterValues() and it didn't seem to work, so I think we can just use `split(",")` to turn it into an array.